test/acceptance/cli-test.js

```diff
@@ -725,14 +725,14 @@ describe('ember-template-lint executable', function () {
           shell: true,
         });
 
-        expect(result.exitCode).toEqual(1);
+        expect(result.exitCode).toEqual(0);
         expect(result.stdout).toBeTruthy();
         expect(result.stderr).toBeFalsy();
       });
     });
 
     describe('given /dev/stdin path', function () {
-      it('should print errors', async function () {
+      it('should not print errors', async function () {
```

Comment:
Not sure what this test is about and why `result.exitCode` is now `0`.

Reply:
The rule used for this test isn't fixable, why should the exit code change?